src/BaGet.Database.PostgreSql/BaGet.Database.PostgreSql.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.0" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.1.2" />
```

Comment:
This package upgrade is necessary as npgsql's identity migration is broken on v3.1.0. This was fixed in 3.1.1.1.